src/Language/PureScript/Pretty/Types.hs

```diff
@@ -124,7 +124,7 @@ matchTypeAtom tro@TypeRenderOptions{troSuggesting = suggesting} =
     typeLiterals = mkPattern match where
       match TypeWildcard{} = Just $ text "_"
       match (TypeVar var) = Just $ text $ T.unpack var
-      match (TypeLevelString s) = Just $ text $ T.unpack $ prettyPrintString s
+      match (TypeLevelString s) = Just $ text $ T.unpack $ prettyPrintLabel $ Label s
```

Comment:
With the changes to `toTypelevelString`, you can revert this. This is definitely not desirable 😄